<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createProjectDialog</class>
 <widget class="QDialog" name="createProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sewers Create Project</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../sewersIcon.ico</normaloff>../sewersIcon.ico</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="createProjectLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <property name="spacing">
      <number>13</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="projectNameLabel">
       <property name="text">
        <string>Project Name: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="projectNameLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Enter a project name...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pipelineTemplateLabel">
       <property name="text">
        <string>Pipeline Template: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="pipelineTemplateComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Asset ESMA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pipeline ESMA</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="browseProjectLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Choose a project directory...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="browseProjectButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="createProjectLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SEWERS CREATE PROJECT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QDialogButtonBox" name="createProjectButtonBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>createProjectButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>createProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createProjectButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>createProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
